### PR TITLE
update docs/app-dev/abci-cli.md

### PR DESCRIPTION
**Pull request by best-jie**
* [x] Updated all relevant documentation in docs
* [ ] Updated all code comments where relevant
* [ ] Wrote tests
* [ ] Updated CHANGELOG_PENDING.md

### CODE REVIEW

File: docs/app-dev/abci-cli.md [hunk 1]
```diff
@@ -327,10 +327,18 @@ application easily in any language.
 We have implemented the counter in a number of languages [see the
 example directory](https://github.com/tendermint/tendermint/tree/develop/abci/example).
 
-To run the Node JS version, `cd` to `example/js` and run
+To run the Node JS version, You'll also need to fetch the relevant repository, from [here](https://github.com/tendermint/js-abci), then install it:
```
Comment (melekes) [inline docs/app-dev/abci-cli.md]:
```suggestion
To run the Node.js version, fist download & install [the Javascript ABCI server](https://github.com/tendermint/js-abci):
```

File: docs/app-dev/abci-cli.md [hunk 2]
```diff
@@ -327,10 +327,18 @@ application easily in any language.
...
 
 ```
-node app.js
+git clone https://github.com/tendermint/js-abci.git
+cd js-abci
+npm install abci
+```
+
+Now run the app:
```
Comment (melekes) [inline docs/app-dev/abci-cli.md]:
```suggestion
Now you can start the app:
```

**Comment by ebuchman** [comment]
Can you update the settings in your fork to allow us to commit directly so we can apply the suggested changes? Thanks!

**Comment by best-jie** [comment]
I have updated the changes of my fork.

**Pull request merged**